ec3730/Data Feeds/WhoisXML/WhoisXMLService.swift

```diff
@@ -199,7 +199,12 @@ class WhoisXMLService: Service {
         guard let endpoint = endpoint(userData), let endpointURL = endpoint.url else {
             throw DataFeedError.invalidUrl
         }
-
+        print("endpoint-----------")
```

Comment:
remove please

Reply:
Ok sure, we were checking the flow and endpoint